src/components/HistoryTable/HistoryTableContext.tsx

```diff
@@ -39,7 +44,7 @@ interface HistoryTableState {
   previousPage: () => void;
   setHistoryTableFilters: (filters: TestFilter[]) => void;
   setSelectedCommit: (order: number) => void;
-  toggleRowSizeAtIndex: (index: number, numCommits: number) => void;
+  toggleFoldedRowExpandedState: (rowIndex: number, expanded: boolean) => void;
```

Comment:
[nit] "state" feels implied
```suggestion
  toggleFoldedRowExpanded: (rowIndex: number, expanded: boolean) => void;
```